packages/web-api/src/controllers/batch-scan-result-controller.ts

```diff
@@ -26,9 +27,13 @@ export class BatchScanResultController extends BaseScanResultController {
 
     public async handleRequest(): Promise<void> {
         const payload = this.tryGetPayload<ScanBatchRequest[]>();
+        if (payload === undefined) {
+            return;
+        }
+
         if (isEmpty(payload)) {
             this.context.res = {
-                status: 422, // Unprocessable Entity,
+                status: 204, // No Content
```

Comment:
This if and the one just above it, can be moved to WebController? I am assuming we need to do this for all type of APIController.

Reply:
Handled as part of ApiController